frontend/packages/container-security/src/kebab-actions.ts

```diff
@@ -0,0 +1,36 @@
+import { K8sKind, PodKind } from '@console/internal/module/k8s';
+import { referenceForModel } from '@console/internal/module/k8s/k8s';
+import { KebabAction, KebabOption } from '@console/internal/components/utils/kebab';
+import { PodModel } from '@console/internal/models';
+import * as _ from 'lodash';
+import { ImageManifestVulnModel } from './models';
+
+const listPathFor = (namespace: string, imageID: string) =>
+  [
+    '/k8s',
+    namespace === '' ? 'all-namespaces' : `ns/${namespace}`,
+    referenceForModel(ImageManifestVulnModel),
+    `?name=sha256.${imageID.split('sha256:')[1]}`,
+  ].join('/');
```

Comment:
Maybe

```ts
`${resourceListPathFromModel(ImageManifestVulnModel, namespace)}?name=sha256.${imageID.split('sha256:')[1]}`;

Reply:
Importing `resourceListPathFromModel` causes circular dependency issue :(

Reply:
OK, no worries. We should track that down :/